assignments/object-functions/code.js

```diff
@@ -0,0 +1,41 @@
+const getX = (object) => { return object.x }
+const point = (x, y) => {
+  let point = { 'x': x, 'y': y }
+  return point
```

Comment:
When you have two lines like this that define a variable and then immediately return its value you can just return the value:

```javascript
  return { 'x': x, 'y': y }
```

Also, when the property names are simple, like there are here you can leave out the quote marks:

```javascript
  return { x: x, y: y }
```